Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -78,75 +108,514 @@ public static void ShowWindow()
+#endif // UNITY_2019_3_OR_NEWER
+                + "\n\n\nYou may also skip this step and configure manually later.", EditorStyles.wordWrappedLabel);
+            CreateSpace(15);
+            using (new EditorGUILayout.HorizontalScope())
+            {
+#if UNITY_2019_3_OR_NEWER
+                if (GUILayout.Button("Legacy XR (Recommended)"))
+#else
+                if (GUILayout.Button("Enable Legacy XR"))
+#endif // UNITY_2019_3_OR_NEWER
+
+                {
+                    XRSettingsUtilities.LegacyXREnabled = true;
+                }
+#if UNITY_2019_3_OR_NEWER
+                if (GUILayout.Button("XR SDK/XR Management"))
+                {
+                    if (!XRSettingsUtilities.XRManagementPresent)
+                    {
+                        AddUPMPackage("com.unity.xr.management");
+                    }
+                    currentStage = ConfigurationStage.InstallBuiltinPlugin;
+                    Repaint();
+                }
+#endif // UNITY_2019_3_OR_NEWER
+                if (GUILayout.Button("Learn more"))
+                {
+                    Application.OpenURL(XRPipelineDocsUrl);
+                }
+            }
+            RenderSetupLaterSection(true, () => {
+                currentStage = ConfigurationStage.ProjectConfiguration;
+                Repaint();
+            });
         }
 
-        private void CompilationPipeline_assemblyCompilationStarted(string obj)
+        private void RenderLegacyXRPipelineDetected()
         {
-            // There should be only one pop-up window which is generally tracked by IsOpen
-            // However, when recompiling, Unity will call OnDestroy for this window but not actually destroy the editor window
-            // This ensure we have a clean close on recompiles when this EditorWindow was open beforehand
-            Close();
+            EditorGUILayout.LabelField("XR Pipeline Setting - LegacyXR in use", EditorStyles.boldLabel);
+            EditorGUILayout.LabelField("To build applications for AR/VR headsets you need to specify an XR pipeline. "
+                + $"\n\nThe LegacyXR pipeline is detected in the project. Please be aware that the LegacyXR pipeline is deprecated in Unity 2019 and is removed in Unity 2020."
+                + $"\n\nFor more information on alternative pipelines, please click on the Learn more button.", EditorStyles.wordWrappedLabel);
+            CreateSpace(15);
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                if (GUILayout.Button("Next"))
+                {
+                    currentStage = ConfigurationStage.ProjectConfiguration;
+                    Repaint();
+                }
+                if (GUILayout.Button("Learn more"))
+                {
+                    Application.OpenURL(XRPipelineDocsUrl);
```

Comment:
Clicking on this button didn't open a URL for me in any of my browsers.

Reply:
That is expected as the page has yet to be created... That will be a separate PR in a separate repo.

Reply:
Is there a way we can alter the button behavior to make it clearer to users that the page is not yet available?

Reply:
Just updated the string to point it to a page on our MR docs site describing different Unity versions and choices of plugins. I will update the link to the new MRTK docs page once it gets published.